data/human/hails.txt

```diff
@@ -5824,6 +5827,279 @@ phrase "friendly deep"
 		"piracy"
 		"social regression"
 		"corruption"
+		"hostility and war"
+		"instability"
+		"anarchism"
+		"hostility"
+		"base values"
+	word
+		"."
+
+phrase "friendly deep"
+	word
+		"We"
+		"Our ships"
+		"Our security forces"
+		"Our fleets"
+		"Our forces"
+	word
+		" "
+	word
+		"" 3
+		"are called to "
+		"are diligent to "
+		"must constantly "
+		"diligently "
+		"always "
+		"must "
+		"shall "
+	word
+		"safeguard"
+		"defend"
+		"protect"
+		"guard"
+		"shield"
+		"advance"
+		"work in advancing"
+	word
+		" "
+	word
+		"" 3
+		"the Deep's "
+		"those who bring "
+		"the ones who serve "
+		"our shared "
+	word
+		""
+		"valued "
+		"invaluable "
+		"hard-fought "
+		"hard-gained "
+	word
+		"democracy"
+		"commonwealth"
+		"sovereignty"
+		"autonomy"
+		"prosperity"
+		"peace"
+		"economic prosperity"
+		"economic progress"
+		"progress"
+		"scientific progress"
+		"social cohesion"
+		"peace, prosperity, and progress"
+		"peace among the stars"
+	word
+		"."
```

Comment:
Apologies for the double comment. I found a cleaner solution after posting that solves the same issue: the phrase "our shared valued democracy" requires a comma after "shared" while the phrase "our shared democracy" doesn't.
```suggestion
		"our shared% "
	word
		""
		"%valued "
		"%invaluable "
		"%hard-fought "
		"%hard-gained "
	word
		"democracy"
		"commonwealth"
		"sovereignty"
		"autonomy"
		"prosperity"
		"peace"
		"economic prosperity"
		"economic progress"
		"progress"
		"scientific progress"
		"social cohesion"
		"peace, prosperity, and progress"
		"peace among the stars"
	word
		"."
	replace
		"% %" ", "
		"% " ""
```